caps.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const events = require('./events');
+
+require('./Modules/vendor');
+require('./Modules/driver');
+
+function eventLogger(eventName, payload) {
+    let currentTimestamp = Math.floor(Date.now() / 1000);
+    console.log('EVENT { ' + 'event: ' + '\'' + eventName + '\',\n' + 'time: ' + currentTimestamp + ',\n' +  ' payload: ' + JSON.stringify(payload) + '}');
+}
+
+events.on('pickup', (pickupPayload) => eventLogger('pickup', pickupPayload));
+
+events.on('in-transit', (transitPayload) => eventLogger('in-transit', transitPayload));
+
+events.on('delivered', (deliveryPayload) => eventLogger('delivered', deliveryPayload));
```

Comment:
Everything is looking good. I would like to see more added to that README, here is the general formatting I follow for that:

```
# Project Name Here

<---- Brief description of the project ---->

## Author: <--- Your name here --->

## Installation

<--- Let people know the steps to getting everything set up --->

### Tests

How to run tests

### UML

<--- UML image here --->

#### Acknowledgments 

<--- People you collaborated with go here, websites that you used possibly --->
```